java/pom.xml

```diff
@@ -148,6 +149,37 @@
       <version>2.0.7.Final</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-server</artifactId>
+      <version>${jetty.version}</version>
+    </dependency>
+
+
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-servlet</artifactId>
+      <version>${jetty.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
+      <version>3.1.0</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.webjars</groupId>
+      <artifactId>bootstrap</artifactId>
+      <version>4.1.1</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.github.spullara.mustache.java</groupId>
```

Comment:
Not knowing anything about java web development, can you comment on the decision to use mustache and bootstrap here?

Reply:
To be honest, I don't have much to say either. I needed a minimalistic server-side templating engine with ~readable syntax. I used Mustache.js a while ago on the client-side, so I looked up a Java port. 

There're other alternatives, e.g:
* [Freemarker](https://freemarker.apache.org/),
* [Jtwig](http://jtwig.org/) which seems to be close to Django/Jinja2 wrt syntax.

Personally, I like Jtwig more than Mustache, but I'd go with whatever library you find more appropriate.

Reply:
I've drafted a Jtwig version and tbh it is not much nicer than the Mustache one. I can push the commit to this branch if you'd like to compare.